lib/cucumber/api/scenario.js

```diff
@@ -23,9 +23,9 @@ var Scenario = function (astTreeWalker, astScenario) {
 
         data.on('data', function(chunk) {
           buffers.push(chunk);
-        })
+        });
         data.on('end', function() {
-          astTreeWalker.attach(Buffer.concat(buffers).toString(), mimeType);
+          astTreeWalker.attach(Buffer.concat(buffers).toString('binary'), mimeType);
```

Comment:
What do you thing about using the `hex` encoding since the `binary` encoding appears to be deprecated?